src/server/db_slice.cc

```diff
@@ -597,23 +597,29 @@ pair<int64_t, int64_t> DbSlice::ExpireParams::Calculate(int64_t now_ms) const {
 
 OpResult<int64_t> DbSlice::UpdateExpire(const Context& cntx, PrimeIterator prime_it,
                                         ExpireIterator expire_it, const ExpireParams& params) {
+  constexpr uint64_t kPersistValue = 0;
   DCHECK(params.IsDefined());
   DCHECK(IsValid(prime_it));
+  // If this need to persist, then only set persist value and return
+  if (params.persist) {
+    UpdateExpire(cntx.db_index, prime_it, kPersistValue);
```

Comment:
I would like to deprecate this UpdateExpire function, specifically please use here RemoveExpire.

Reply:
OK